test/test.js

```diff
@@ -158,6 +158,46 @@ describe('remapify', function(){
     }])
   })
 
+  it('looks for files in multiple directories, in precedence', function(done){
+    b.on('remapify:files', function(files, expandedAliases){
+      expandedAliases.should.contain.keys(
+        'a.js',
+        'b.js'
+      )
+      expandedAliases['a.js'].split(path.sep).join('/').should.equal('./test/fixtures/target2/a.js')
+      expandedAliases['b.js'].split(path.sep).join('/').should.equal('./test/fixtures/target/b.js')
+
+      b.emit.should.not.have.been.calledWith('error')
+
+      done()
+    })
+
+    plugin(b, [{
+      src: '**/*.js'
+      , cwd: ['./test/fixtures/target2', './test/fixtures/target']
+    }])
+  })
+
+  it('multiple cwds works when passing in an array from command line', function(done){
+    b.on('remapify:files', function(files, expandedAliases){
+      expandedAliases.should.contain.keys(
+        'a.js',
+        'b.js'
+      )
+      expandedAliases['a.js'].split(path.sep).join('/').should.equal('./test/fixtures/target2/a.js')
+      expandedAliases['b.js'].split(path.sep).join('/').should.equal('./test/fixtures/target/b.js')
+
+      b.emit.should.not.have.been.calledWith('error')
+
+      done()
+    })
+
+    plugin(b, [{
+      src: '**/*.js'
+      , cwd: {'_': ['./test/fixtures/target2', './test/fixtures/target'] }
```

Comment:
Huh. That's gotta be coming from minimist, and is likely a bug in browserify. You might want to submit an issue there and reference that in the code comment.

If we're going to support this hack, mind adding a test that shows that just an array works too?